packages/p2p/src/components/floating-rate/floating-rate.scss

```diff
@@ -0,0 +1,177 @@
+            opacity: 1;
+        }
+        input::-moz-placeholder {
+            /* Mozilla Firefox 19+ */
+            color: var(--text-less-prominent) !important;
+            opacity: 1;
+        }
+        input:-ms-input-placeholder {
+            /* Internet Explorer 10-11 */
+            color: var(--text-less-prominent) !important;
+        }
+        input::-ms-input-placeholder {
+            /* Microsoft Edge */
+            color: var(--text-less-prominent) !important;
+        }
+
+        &:hover {
+            border-color: var(--border-hover);
+        }
+        &:active,
+        &:focus {
+            border-color: var(--border-active);
+        }
+
+        &__error_field {
+            color: $color-red-1;
+            border-color: $color-red;
+        }
+    }
+
+    &__percent {
+        position: absolute;
+        height: 3.2rem;
+        right: 4rem;
+        align-items: center;
+        justify-content: center;
+        display: flex;
+        background: transparent;
+        border-color: transparent;
+        z-index: 2;
+        color: inherit;
+
+        &--symbol {
+            font-size: 1.4rem;
+            line-height: 1.5;
+            padding-top: 0.7rem;
+        }
+        &:before {
+            @include typeface(--paragraph-center-normal-black);
+            color: inherit;
+        }
+    }
+
+    &__mkt-rate {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: flex-start;
+        padding: 2px 8px;
+
+        position: static;
+        height: 40px;
+        left: 160px;
+        top: 0px;
+
+        background: #377cfc14;
+        border-radius: 0px 4px 4px 0px;
+
+        /* Inside auto layout */
+
+        flex: none;
+        order: 1;
+        align-self: stretch;
+        flex-grow: 1;
+
+        &__label {
+            position: static;
+
+            /* desktop/extra small/XS - regular */
+
+            font-style: normal;
+            font-weight: normal;
+            font-size: 10px;
+            line-height: 14px;
+            /* identical to box height, or 140% */
+
+            /* light / 2 - general (text) */
+
+            color: $color-black-1;
+
+            /* Inside auto layout */
+
+            flex: none;
+            order: 1;
+            flex-grow: 0;
+            margin: 0px;
+        }
+
+        &__msg {
+            position: static;
+            height: 18px;
+            left: 8px;
+            top: 18px;
+
+            /* desktop/small/S - regular */
+
+            font-style: normal;
+            font-weight: normal;
+            font-size: 12px;
+            line-height: 18px;
+
+            color: $color-black-1;
+
+            /* Inside auto layout */
+
+            flex: none;
+            order: 2;
+            flex-grow: 0;
+            margin: 0px;
+        }
+    }
+
+    &__hint {
+        padding-left: 1rem;
+        font-size: 1.2rem;
+        color: $color-blue-4;
+        font-weight: 400;
```

Comment:
Please use the `Text` component to set font size, font weight, line height and font color.